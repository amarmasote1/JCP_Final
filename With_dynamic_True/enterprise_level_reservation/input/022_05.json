[
  {
    "storeID": "1223",
    "itemID": "17345670022",
    "uom": "EACH",
    "supplyData": {
      "onhandsupply": 50
    },
    "demandData": {
        "reserveddemand": 0
      },
    "timestamp": "<$currentTime>"
    
  }
]