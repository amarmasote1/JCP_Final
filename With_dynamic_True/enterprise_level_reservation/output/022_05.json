{
    "message": "Success",
    "body": [
        {
            "itemID": "17345670022",
            "uom": "EACH",
            "demandData": {
                "reserveddemand": 0
            },
            "markDown": {
                "active": false
            },
            "storeID": "1223",
            "supplyData": {
                "onhandsupply": 50
            },
            "timestamp": "2020-02-28 12:45:54.0000 IST"
        }
    ],
    "status": 200
}