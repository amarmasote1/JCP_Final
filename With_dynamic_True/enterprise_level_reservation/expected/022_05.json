{"status":200,"message":"Success","body":[{"itemID":"17345670022","uom":"EACH","storeID":"1223","supplyData":{"onhandsupply":50},"demandData":{"reserveddemand":0}}]}